src/Command/Serve.php

```diff
@@ -34,17 +35,32 @@ final class Serve extends Command
     protected static $defaultName = 'serve';
     protected static $defaultDescription = 'Runs PHP built-in web server';
 
+    public function __construct(private Aliases $aliases)
+    {
+        parent::__construct();
+    }
+
     public function configure(): void
     {
         $this
             ->setHelp('In order to access server from remote machines use 0.0.0.0:8000. That is especially useful when running server in a virtual machine.')
-            ->addArgument('address', InputArgument::OPTIONAL, 'Host to serve at', 'localhost')
+            ->addArgument('address', InputArgument::OPTIONAL, 'Host to serve at', '127.0.0.1')
             ->addOption('port', 'p', InputOption::VALUE_OPTIONAL, 'Port to serve at', self::DEFAULT_PORT)
             ->addOption('docroot', 't', InputOption::VALUE_OPTIONAL, 'Document root to serve from', self::DEFAULT_DOCROOT)
             ->addOption('router', 'r', InputOption::VALUE_OPTIONAL, 'Path to router script', self::DEFAULT_ROUTER)
             ->addOption('env', 'e', InputOption::VALUE_OPTIONAL, 'It is only used for testing.');
     }
 
+    public function complete(CompletionInput $input, CompletionSuggestions $suggestions): void
+    {
+        if ($input->mustSuggestArgumentValuesFor('address')) {
+            $suggestions->suggestValues(['localhost', '127.0.0.1', '0.0.0.0']);
+            return;
+        }
+
+        $suggestions->suggestOptions($this->getDefinition()->getOptions());
```

Comment:
Seems, autocompletion good work without this line.

Reply:
Yes, checked.